Frameworks/GoSSB.xcframework/ios-arm64/Headers/GoSSB.h

```diff
@@ -68,4 +68,8 @@ extern bool ssbBlobsWant(gostring_t ref);
 extern int ssbBlobsGet(gostring_t ref);
 extern char* ssbBlobsAdd(int32_t fd);
 
+extern char* ssbRoomsListAliases(gostring_t address);
+extern bool ssbRoomsAliasRegister(gostring_t address, gostring_t alias);
```

Comment:
@boreq can you return the aliasURL here instead of a bool? That will let me store the alias in the db right after this call succeeds without needing to call ssbRoomsListAliases.

Reply:
Can you check the new signature now?